moped-database/migrations/1694199966743_more_entities/up.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO moped_entity (entity_name, organization_id, department_id) values
+    ('City of Austin (COA)', 1, 1),
+    ('University of Texas', 4, null),
+    ('Austin Transit Partnership (ATP)', 1, null);
```

Comment:
i squeezed these in here bc i had PR fatigue at the moment. but i don't mind separating this out.